plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,632 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024, IamLunchbox <r.grieger@hotmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+author: "Raphael Grieger (@IamLunchbox) <r.grieger@hotmail.com>"
+short_description: Start a VM backup in Proxmox VE cluster
+version_added: 10.0.0
+description:
+  - Allows you to create backups of KVM and LXC guests in Proxmox VE cluster.
+  - Offers the GUI functionality of creating a single backup as well as using the run-now functionality from the cluster backup schedule.
+  - The mininum required privileges to use this module are VM.Backup and Datastore.AllocateSpace.
+  - Most options are optional and if unspecified will be chosen by the Cluster and its default values.
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+  action_group:
+    version_added: 10.0.0
+options:
+  backup_mode:
+    description:
+      - The mode how proxmox performs backups. The default is, to create a runtime snapshot including memory.
```

Comment:
The product Proxmox is written with a capital P. Please check other places in the documentation to make sure it's always spelled correctly.
```suggestion
      - The mode how Proxmox performs backups. The default is, to create a runtime snapshot including memory.
```